Python/specialize.c

```diff
@@ -1914,8 +1905,7 @@ _Py_Specialize_BinaryOp(PyObject *lhs, PyObject *rhs, _Py_CODEUNIT *instr,
             }
             if (PyUnicode_CheckExact(lhs)) {
                 _Py_CODEUNIT next = instr[INLINE_CACHE_ENTRIES_BINARY_OP + 1];
-                bool to_store = (next.op.code == STORE_FAST ||
-                                 next.op.code == STORE_FAST__LOAD_FAST);
+                bool to_store = (next.op.code == STORE_FAST);
```

Comment:
Shouldn't we still be able to support "next opcode is `STORE_FAST_LOAD_FAST`" here?

Reply:
Nah, because then the `BINARY_OP_INPLACE_ADD_UNICODE` instruction would have to do both the store *and* the load.